src/elections/tables.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
+
+from constants import (
+    COMPUTING_ID_LEN,
+    DISCORD_ID_LEN,
+    DISCORD_NAME_LEN,
+    DISCORD_NICKNAME_LEN,
+)
+from database import Base
+from sqlalchemy import (
+    Column,
+    DateTime,
+    ForeignKey,
+    PrimaryKeyConstraint,
+    String,
+    Text,
+)
+
+
+# Each row represents an instance of an
+class Election(Base):
+    __tablename__ = "election"
+
+    # Slugs are unique identifiers
+    slug = Column(String(32), primary_key=True)
+    name = Column(String(32), nullable=False)
+    officer_id = Column(String(COMPUTING_ID_LEN), nullable=False)
+    # Can be one of (general_election: General Election, by_election: By-Election, council_rep_election: Council Rep Election)
+    type = Column(String(64), default="general_election")
+    date = Column(DateTime, nullable=False)
+    end_date = Column(DateTime)
+    websurvey = Column(String(300))
```

Comment:
rip websurvey. Is this supposed to store a websurvey link? Since websurvey is dead now, maybe we should just make it `survey_link`?